src/aiida/orm/nodes/data/array/projection.py

```diff
@@ -278,7 +278,7 @@ def array_list_checker(array_list, array_name, orb_length):
                 raise exceptions.ValidationError('Tags must set a list of strings')
             self.base.attributes.set('tags', tags)
 
-    def set_orbitals(self, **kwargs):
+    def set_orbitals(self, **kwargs):  # type: ignore[override]
```

Comment:
I'll just leave `kwargs` as it is and ignore type checking.

Reply:
Can you add a TODO comment with the error? The mypy error suggests that this is an actual error? Or do you think it's a false positive?

```python
src/aiida/orm/nodes/data/array/projection.py:281: error: Signature of "set_orbitals" incompatible with supertype "OrbitalData"  [override]
src/aiida/orm/nodes/data/array/projection.py:281: note:      Superclass:
src/aiida/orm/nodes/data/array/projection.py:281: note:          def set_orbitals(self, orbitals: Any) -> Any
src/aiida/orm/nodes/data/array/projection.py:281: note:      Subclass:
src/aiida/orm/nodes/data/array/projection.py:281: note:          def set_orbitals(self, **kwargs: Any) -> Any
```

Reply:
Can we just merge it and if this is a problem make an issue out of it?